LOAN/LoanContracts/LoanCore.rdf

```diff
@@ -1798,11 +1720,27 @@ As needed, use the worksFor property to connect appraisers to the appraisal mana
 		<fibo-fnd-utl-av:adaptedFrom>the 2015 Revised HMDA regulation.</fibo-fnd-utl-av:adaptedFrom>
 	</owl:DatatypeProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-loan-ln-ln;hasOriginatorCompany">
-		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;hasParty"/>
-		<rdfs:label>has originator company</rdfs:label>
-		<rdfs:range rdf:resource="&fibo-fnd-org-org;Organization"/>
-		<skos:definition>relates something, typically  a loan contract, to the  company affilliated with the person originating the loan</skos:definition>
+	<owl:ObjectProperty rdf:about="&fibo-loan-ln-ln;hasOriginatingServiceProvider">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;hasPartyInRole"/>
+		<rdfs:label>has originating service provider</rdfs:label>
+		<rdfs:range>
+			<owl:Class>
+				<owl:intersectionOf rdf:parseType="Collection">
+					<rdf:Description rdf:about="&fibo-fbc-pas-fpas;FinancialServiceProvider">
+					</rdf:Description>
+					<owl:Class>
+						<owl:unionOf rdf:parseType="Collection">
+							<rdf:Description rdf:about="&fibo-fnd-agr-ctr;ContractParty">
+							</rdf:Description>
+							<rdf:Description rdf:about="&fibo-fnd-agr-ctr;ContractThirdParty">
+							</rdf:Description>
+						</owl:unionOf>
+					</owl:Class>
+				</owl:intersectionOf>
+			</owl:Class>
+		</rdfs:range>
+		<skos:definition>relates something, typically a loan contract, to the service provider responsible for originated the loan</skos:definition>
```

Comment:
"originated" should be "originating", but it's somewhat circular in leaving "originate" undefined.